src/auction.js

```diff
@@ -303,11 +293,15 @@ export const addBidResponse = createHook('asyncSeries', function(adUnitCode, bid
 
 export function auctionCallbacks(auctionDone, auctionInstance) {
   let outstandingBidsAdded = 0;
-  let doneCalled = false;
+  let allAdapterCalledDone = false;
+
+  let areAllAdapterDone = delayExecution(() => {
```

Comment:
nice usage here. One knit (I know it's not part of this PR) but the name of this function is somewhat confusing. I was expecting it to wait for the next turn of the event loop, but it's synchronous.